python/pyarrow/_dataset.pyx

```diff
@@ -936,7 +936,7 @@ cdef class ParquetFileFragment(FileFragment):
 
         Parameters
         ----------
-        predicate : Expression, default None
+        filter : Expression, default None
             Exclude RowGroups whose statistics contradicts the predicate.
```

Comment:
Update this sentence? Perhaps "Only include those row groups which satisfy this predicate"